force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -46,7 +55,8 @@ public class CollectionsTasks {
       if (setFromList.contains(sum - element)) {
         result = true;
         break;
-      }
+      } else
+        result = false;
```

Comment:
No, it's not that easy :)

Let's take a look at a more complex example:

```
Boolean result = CollectionsTasks.sumCanBeReachedOptimal(0, new List<Integer>{12, -3, 10, 1, 0});
System.debug('Should be false. Actual: ' + result);
```

 This also doesn't work

Reply:
We will probably update the test with these examples... Cause other students have tried to hack the solution but faced similar issues

Reply:
fixed by removing the element itself from checking .contains

Reply:
👍